lib/datasets/pascal_voc.py

```diff
@@ -12,47 +12,49 @@
 import numpy as np
 import scipy.sparse
 import scipy.io as sio
-import utils.cython_bbox
 import cPickle
 import subprocess
 import uuid
 from voc_eval import voc_eval
 from fast_rcnn.config import cfg
 
+
 class pascal_voc(imdb):
-    def __init__(self, image_set, year, devkit_path=None):
-        imdb.__init__(self, 'voc_' + year + '_' + image_set)
+    def __init__(self, image_set, year, devkit_path=None, class_set=None):
+        if class_set is None:
+            class_set = imdb.USE_PASCAL_CLASSES
+        imdb.__init__(self, 'voc_' + year + '_' + image_set, class_set)
         self._year = year
         self._image_set = image_set
         self._devkit_path = self._get_default_path() if devkit_path is None \
-                            else devkit_path
+            else devkit_path
         self._data_path = os.path.join(self._devkit_path, 'VOC' + self._year)
-        self._classes = ('__background__', # always index 0
-                         'aeroplane', 'bicycle', 'bird', 'boat',
-                         'bottle', 'bus', 'car', 'cat', 'chair',
-                         'cow', 'diningtable', 'dog', 'horse',
-                         'motorbike', 'person', 'pottedplant',
-                         'sheep', 'sofa', 'train', 'tvmonitor')
-        self._class_to_ind = dict(zip(self.classes, xrange(self.num_classes)))
+        # self._classes = ('__background__', # always index 0
+        #                  'aeroplane', 'bicycle', 'bird', 'boat',
+        #                  'bottle', 'bus', 'car', 'cat', 'chair',
+        #                  'cow', 'diningtable', 'dog', 'horse',
+        #                  'motorbike', 'person', 'pottedplant',
+        #                  'sheep', 'sofa', 'train', 'tvmonitor')
+        # self._class_to_ind = dict(zip(self.classes, xrange(self.num_classes)))
```

Comment:
Because this code has been moved elsewhere, it probably makes more sense to delete these lines than to comment them out.